Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:20-alpine
 
-ENV EXTERNAL_PORT 3000
-ENV PORT 3000
+ENV EXTERNAL_PORT 3500
+ENV PORT 3500
```

Comment:
I think you should revert these - we can override them with .env when use `docker compose`